admin/views/dashboard-widget.php

```diff
@@ -13,15 +13,15 @@
  */
 
 ?>
-<p><?php _e( 'Below are your published posts&#8217; SEO scores. Now is as good a time as any to start improving some of your posts!', 'wordpress-seo' ); ?></p>
+<p><?php printf( __( 'Hey %s,', 'wordpress-seo' ), $user->user_firstname ); ?></p>
```

Comment:
How will this look if the profile has no first name? A check should probably look to see if this is empty before using it.